src/rules/converters/no-null-keyword.ts

```diff
@@ -0,0 +1,12 @@
+import { RuleConverter } from "../converter";
+
+export const convertNoNullKeyword: RuleConverter = () => {
+    return {
+        rules: [
+            {
+                ruleName: "no-null/no-null",

```

Comment:
This rule actually has a bit of a behavioral change from TSLint to ESLint. The ESLint rule doesn't handle `null` types:

```ts
let wat: string | null = null;
```

Could you please add a `notices` array explaining that?